### DISCUSSION TITLE
面對不同對象使用不同的通訊軟體

### DISCUSSION THREAD

**Discussion by curtis1017** [1101-GS4719-C/Python-Community-GS4719-C#1243] [General]
現代人每天都會使用到的通訊工具中，Line 與 IG（Instagram）是被認為較受歡迎也較為普及的兩個軟體，不久前日本就有電視節目針對 1990 年代中後期至2010 年代初期出生的「Z世代」調查他們習慣使用的手機 APP，發現 IG 成了日本年輕人最常使用的通訊軟體，只有相處到一定程度、覺得夠熟的朋友，才會交換彼此的 LINE。在這個地方，想要提醒大家，如果不是為了作業上，異性給妳他自己的LINE，帶鰾你是有機會能跟他成為非常熟的朋友。
再來，最近日本電視台晨間資訊節目《ZIP!》到了街頭隨機採訪了一些年輕人，調查中發現一個相當有趣的趨勢，原來有許多年輕人都把 IG 當成了主要的即時通訊軟體，平均下來一天當中有將近一半到四分之一的時間都在使用IG，而 LINE 的使用量則是遠遠落後了。不過這並不代表年輕人捨棄了 LINE ，而是這兩款即時通訊軟體在他們心中有著明顯的分工，會被用來與不同定位的人做聯繫；根據受訪的年輕人表示，當他們與剛認識的新朋友交換聯絡方式時，通常會更傾向於交換 IG 而不是 LINE，這是因為可以藉由 IG 的限時動態、發文來了解彼此的喜好或性格，就算沒有互動也沒關係，還可以有限度地拉開與不熟的人的距離，也可以藉由觀察來決定要不要跟對方深交，他們會認為使用 IG 聯繫的感受會較為輕鬆。

**Comment by zilinng** [comment]
我自己面對不同的人也習慣用不一樣的通訊軟體，但卻不是跟日本人一樣用關係親疏區別，單純只是別人比較常用哪個軟體回我我就會逐漸習慣也只用那個軟體找他，也有朋友一下用ig 一下用line 那我們就會同時在兩個軟體聊天。

**Comment by Evening218** [comment]
就我自己的使用經驗而言，ig比較可能追蹤的是現實不一定有交流的人，而line就是用來跟已經認識的或是比較親近的人聯絡用，另外就是line傳檔案比ig方便很多。

**Comment by Chiuhsianglin** [comment]
我一直都認為ig不是一個可以固定跟別人聊天的平台，我會習慣用line，ig只是觀看或回別人限時動態，小聊一下的地方。

**Comment by ziihui** [comment]
Ig感覺比較像是在聯繫分散各地的好朋友～
Line的話就是用來跟聯繫關係更上一層的家人朋友！
那工作、報告的話用message最棒～！
用賴的話看到已讀不回就會很躁＞＜
Message就能馬上知道誰讀了訊息！
溝通上比較方便快速～

**Comment by llh26** [comment]
關於通訊軟體，我自己也有小觀察，我觀察到不同性別的人，對通訊軟體的偏好似乎會有所不同。以我身邊的人為例，女生通常偏好使用LINE和IG，我們（女性）通常不會把Messenger當作第一選項，而男生通常偏好Messenger或IG，我訪問過身邊的男性友人，發現他們覺得Messenger比較方便，男生圈裡大家也常用這個，自然而然就繼續用下去，不知道大家是否也這麼覺得～

**Comment by pigeon9975** [comment]
我也會針對不同對象使用不同通訊軟體！像對高中同學我普遍喜歡用Line，而大學同學則比較喜歡用msg溝通，此外，我覺得msg在做報告等需要小組分工的場合超有用的！可以看誰有讀訊息，如果有人已讀不回或擺爛馬上就可以抓到，超級方便！

**Comment by Jack-1D** [comment]
我認為是每個通訊軟體的功能都有好有壞。像是我就很喜歡iPhone的messenger，可以長按來已讀別人的訊息，但是基於安全問題，就沒辦法傳zip檔，也不能傳太長的訊息。
